Fibonachi.java

```diff
@@ -0,0 +1,47 @@
+import java.util.Scanner;
+
+public class Fibonachi {
+  public static void main (String[] args) {
+    System.out.println("Please enter a number:");
+    Scanner number = new Scanner(System.in);
+    String string = number.next();
+    int number_int = Integer.parseInt(string);
+    int firstNumber = 1;
+    int secondNumber = 1;
+    int thirdNumber = 1;
+    int countNumberFebonacci = 0;
+    int numberForTest = 0; 
+    boolean resultOfTest = false;
+    if (number_int == 0 || number_int == 1 || number_int == -1) {
+      System.out.println("It's number of sequence Febonachi");
+    } else {
+      if (number_int < 0) {
```

Comment:
You cannot do it as user explicitly said he want to check negative number. He doesn't want to check |negative number|